py/visdom/static/index.html

```diff
@@ -29,12 +29,12 @@
     <script src={{ static_url("js/react-dom.min.js") }}></script>
     <script src={{ static_url("js/layout_bin_packer.js") }}></script>
 
-    <!-- Mathjax -->
-    <script type="text/javascript" async src={{ static_url("js/mathjax-MathJax.js") }}></script>
-
     <!-- Plotly -->
     <script src={{ static_url("js/plotly-plotly.min.js") }}></script>
 
+    <!-- Mathjax -->
+    <script type="text/javascript" async src={{ static_url("js/MathJax.js") }}></script>
+
```

Comment:
It's important that the `MathJax` library is imported _before_ the `Plotly` library, so please don't swap the ordering here.

Reply:
@JackUrb  I do not ant to swap the lines. But what I found I will share with you.
Actually, while I was testing the edited code for mathjax, I have found that Plotly suddenly gave some error and that were logged in the console.
But when I swapped the loading positions of the JS, it was working fine.
But if you think that it is not appropriate then I will make the changes. 
Here take a look at this error in the console when plotly is placed after the Mathjax:
![image](https://user-images.githubusercontent.com/33365560/78382375-64183700-75f4-11ea-9346-6d1dbc012e25.png)

Let me know and I will make the changes if you say so.

Reply:
Ah okay so there's still something weird going on with this import - The entire goal is to be able to use `LaTeX` in Plotly, and `MathJax` is supposed to enable that. When `MathJax` is imported before Plotly, Plotly attempts to integrate with the library.

Reply:
ok .. I got it ... just give me some time and it will be fixed.. Insha allah..